tests/EncryptedFS.concurrent.test.ts

```diff
@@ -23,6 +24,23 @@ describe(`${EncryptedFS.name} Concurrency`, () => {
   let db: DB;
   let iNodeMgr: INodeManager;
   let efs: EncryptedFS;
+
+  const gateFactory = (s: fc.Scheduler) => {
+    const scheduledStarter = s.scheduleFunction(async () => {});
+    return async <T>(f: () => Promise<T>) => {
+      await scheduledStarter();
+      return await f();
+    };
+  };
```

Comment:
We should find a better way to apply barriers to the scheduling system of fast-check. Should search through the issues.

Reply:
I think there is way to do this properly, see: https://github.com/dubzzz/fast-check/issues/564.

And the associated commits.

Reply:
Replaced the `gateFactory` with the `scheduleCall` implementation. This adds the ability to label the calls for clearer test failures.

Reply:
I suggest moving this into `tests/utils`.